src/plugins/ecs/__init__.py

```diff
@@ -479,5 +527,81 @@ def tasks_get_lookup_term(text):
                 return data[(data.lower().find('---') + 3):]
 
 
-
-
+#retrieve json data from super jenkins for build urls
+def get_superjenkins_data(superjenkins_link,beginning_script_tag,ending_script_tag):
+
+    # get json to get individual services/application build links
+    cached_items = ""
+    cached_array = ""
+
+    try:
+        returned_data = requests.get(superjenkins_link)
+        returned_data_iterator = returned_data.iter_lines()
+
+        for items in returned_data_iterator:
+            if beginning_script_tag in items:
+                cached_items = items.replace(beginning_script_tag, "").replace(ending_script_tag, "")
```

Comment:
One thing we will need to check now that we are caching is the memory allocation on the lambda function.  Lambda functions have a max execution time and a memory limit.  It's probably ok but worth reviewing what it is given we are caching the superjenkins response now